src/Youjin.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from "react";
+
+function Youjin() {
+  const [state, setState] = useState();
+  const [code, setCode] = useState(1);
+  console.log("test2");
+
+  const test = "1";
+
+  useEffect(() => {
+    fetch(`http://www.wecode.kr/${code}`)
+      .then((res) => res.json())
+      .then(setState);
+  }, []);
```

Comment:
Codacy has a fix for the issue: [React Hook useEffect has a missing dependency: 'code'. Either include it or remove the dependency array.](https://app.codacy.com/gh/wecode-bootcamp-korea/test-repository/pullRequest?prid=10063662)
```suggestion
  }, [code]);
```